packages/types/lib/logs/messages.ts

```diff
@@ -126,6 +126,8 @@ export interface MessageRow {
               addedKeys: string[];
               updated: number;
               updatedKeys: string[];
+              unchanged: number;
+              unchangedKeys: string[];
```

Comment:
my guess is that a lot of records are actually unchanged so, as you mentioned, we might end up logging way more data. should we start by only adding the unchanged count to the logs, assess the impact once we have the numbers and see what would be the impact before we start adding the unchangedKeys to the logs?